docs/style-guide.adoc

```diff
@@ -526,6 +585,24 @@ RETURN split( 'original', 'i' )
 RETURN split('original', 'i')
 ----
 
+. Use padding space on the inside of simple subquery expressions.
```

Comment:
"Use padding space on the inside of simple subquery expressions."
->
"Use padding space within simple subquery expressions."